packages/integration-tests/projects/suite-web/tests/wallet/notification-toast.test.ts

```diff
@@ -0,0 +1,43 @@
+// @group:suite
+// @retry=2
+
+describe('Check notification toast', () => {
+    beforeEach(() => {
+        cy.task('startBridge');
+        cy.task('startEmu', { version: Cypress.env('emuVersionT1'), wipe: true });
+        cy.task('setupEmu', { needs_backup: false });
+        cy.viewport(1080, 1440).resetDb();
+        cy.prefixedVisit('/settings/device');
+        cy.passThroughInitialRun();
+    });
+
+    afterEach(() => {
+        cy.task('stopEmu');
+    });
+
+    /* Test case
+     * 1. Pass onboarding.
+     * 2. Navigate to Settings/device.
+     * 3. Change passphrase settings.
+     * 4. Confirm on device.
+     * 5. Check that toast notification is rendered.
+     */
+    it('Check notification toast', () => {
+        //
+        // Test preparation
+        //
+        // Test execution
+        //
+        cy.log('turn on passphrase protection');
+        cy.getTestElement('@settings/device/passphrase-switch')
+            .click({ force: true })
+            .getConfirmActionOnDeviceModal();
+        cy.task('pressYes');
+        cy.getConfirmActionOnDeviceModal().should('not.exist');
+        cy.wait(1000);
```

Comment:
wait not generally nice and also not needed here

Reply:
@mroz22 removed